src/MvvmBlazor/ViewModel/ViewModelBase.cs

```diff
@@ -17,6 +22,14 @@ protected bool Set<T>(ref T field, T value, [CallerMemberName] string? propertyN
             {
                 field = value;
                 OnPropertyChanged(propertyName!);
+                if (!_subscriptions.ContainsKey(propertyName))
```

Comment:
Could you fix the warnings for possible null references?

Reply:
Done!